pyne/openmc_utils.py

```diff
@@ -5,7 +5,10 @@
 import io
 import sys
 from warnings import warn
-from collections import namedtuple
+try:
+    from collections.abc import namedtuple
+except ImportError:
+    from collections import namedtuple
```

Comment:
I think this one did't need to be changed. as it does not rely on `abc` in the first place.